xpk.py

```diff
@@ -1964,9 +1969,48 @@ def create_vertex_tensorboard(args) -> map:
                                             tensorboard_name=tensorboard_name)
   if instance_id:
```

Comment:
do we need the # pylint: disable=used-before-assignment on ln 1967?  I can't figure what is it used for!

Reply:
Interestingly, pylint complains about using `tensorboard.create_instance()` before assignment in `create_vertex_tensorboard()` but not in `create_vertex_experiment()`. I think it is because I am installing the required dependency behind a flag:
```
if _VERTEX_TENSORBOARD_FEATURE_FLAG:
  from cloud_accelerator_diagnostics import tensorboard
```

This is to ensure that current xpk users do not have to make sure to install `cloud-accelerator-diagnostics` package until we release this Vertex feature to them.

Reply:
Oh I see, interesting, good to know! Thanks Surbhi